sway-error/src/convert_parse_tree_error.rs

```diff
@@ -9,8 +9,6 @@ pub enum ConvertParseTreeError {
     FunctionArbitraryExpression { span: Span },
     #[error("generics are not supported here")]
     GenericsNotSupportedHere { span: Span },
-    #[error("fully qualified paths are not supported here")]
-    FullyQualifiedPathsNotSupportedHere { span: Span },
```

Comment:
Does this mean I can now write
```
    fn test_function(a: lib1::Foo, b: lib2::Foo) -> bool;
```
? That is, can i use call paths when referring to a type?

Reply:
Oh this seems related https://github.com/FuelLabs/sway/issues/1644

Reply:
Yeah nevermind, I was thinking of a different error here.